ALCF/helpers.sh

```diff
@@ -557,21 +548,27 @@ make_ds_hostfile() {
 }
 
 ezpz_savejobenv() {
-    local outfile="${WORKING_DIR}/bin/savejobenv" ; mkdir -p $(dirname "${outfile}")
-    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${outfile}" && source "${outfile}"
+    file=$(mktemp)
+    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${file}"
+    source "${file}" || exit
```

Comment:
Use `mktemp` to ensure safe temporary file creation.

This function will:

1. download [`ezpz/src/ezpz/bin/savejobenv`](https://github.com/saforem2/ezpz/blob/main/src/ezpz/bin/savejobenv) and save it to `file=$(mktemp)`
2. source `$file`
    - this will, in turn, parse the `PBS_*` variables from the current environment, use this to determine relevant job information (number of nodes, number of GPUs on each node, etc) to build a suitable `alias launch="mpiexec ..."`